virtual_rainforest/models/animals/animal_model.py

```diff
@@ -53,8 +54,11 @@ class AnimalModel(BaseModel):
     """Longest time scale that soil model can sensibly capture."""
     required_init_vars = ()
     """Required initialisation variables for the animal model."""
-    vars_updated = []
-    """The animal model doesn't currently use the data object so nothing to update."""
+    vars_updated = ["excess_excrement"]
```

Comment:
Assuming this value is never changed, I think it would be better as a `tuple`:

```suggestion
    vars_updated = ("excess_excrement",)
```

Reply:
The value should never be changed in the course of a simulation but more elements will be added as the model develops. Is the advantage to use a tuple here that it is immutable?

Reply:
Yes, exactly.